gradle.properties

```diff
@@ -1 +1,5 @@
-scalaVersions=2.12.7, 2.11.12
\ No newline at end of file
+version=0.1.0
+scalaVersions=2.12.7, 2.11.12
+
+nexusSeekOssUsername=
```

Comment:
So this is set on a developer's laptop? Github Actions supports secrets if you wanted to automatically push on merge to master.